sdk/lib/pseh/include/pseh/pseh2_64.h

```diff
@@ -173,3 +173,12 @@ __seh2$$begin_except__: __MINGW_ATTRIB_UNUSED;
 #define _SEH2_LEAVE goto __seh2$$leave_scope__
 #define _SEH2_YIELD(__stmt) __stmt
 #define _SEH2_VOLATILE volatile
+
+#define __try _SEH2_TRY
```

Comment:
No `#ifndef __try` needed in the x64 version ?

Reply:
In fact, yes. It just passed, because we allow warnings on x64 builds. Fixed it.